db/seeds.rb

```diff
@@ -5,3 +5,16 @@
 #
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
+
+  30.times do
+    Giveaway.create(
+    title: Faker::Commerce.product_name,
+    description: Faker::Lorem.paragraphs,
+    location: Faker::Address.city
+    ).comments.create(
+          commenter: Faker::Movies::Lebowski.character,
+          body: Faker::Movies::Lebowski.quote
+        )
```

Comment:
I believe it's better to store creating giveaways in one variable, and then create two comments per giveaway.

Something like:
```
giveaway =  Giveaway.create(
    title: Faker::Commerce.product_name,
    description: Faker::Lorem.paragraphs,
    location: Faker::Address.city
    )

2.times do |c|
  giveaway..comments.create(
          commenter: Faker::Movies::Lebowski.character,
          body: Faker::Movies::Lebowski.quote
        )
end
```

Like this, you have separate logic and more control over it.

Reply:
@violeta-p  How to do that with 10 giveaways with 2 comments each?